src/common/kqueue.c

```diff
@@ -150,7 +150,9 @@ kqueue(void)
     tracing_mutex_init(&kq->kq_mtx, NULL);
 
     if (kqops.kqueue_init(kq) < 0) {
+    error:
         free(kq);
+        tracing_mutex_destroy(&kq->kq_mtx);
```

Comment:
Destroying the mutex _AFTER_ freeing the kq?

Reply:
:exploding_head: It was a _typo_ :laughing: . fixed